ui/src/data_explorer/components/SendToDashboardOverlay.tsx

```diff
@@ -279,8 +303,14 @@ class SendToDashboardOverlay extends PureComponent<Props, State> {
 
     let selectedDashboards = this.selectedDashboards
 
-    if (_.includes(selectedIDs, NEW_DASHBOARD_ID)) {
-      const {data} = await createDashboard(NEW_EMPTY_DASHBOARD)
+    if (this.isNewDashboardSelected) {
+      const {data} =
+        newDashboardName === ''
```

Comment:
it seems like if a user de-selects Send to New Dashboard, the state value for `newDashboardName` will be retained, and thus this logic will make it send to a new one anyway. is that correct? would you make it to that it is reset when de-selected, if so?

Reply:
hm I just tested it and it doesn't seem to make a new dashboard if you deselect "Send to a New Dashboard." 

I didn't change the logic for whether or not a new dashboard gets created--it used the "selectedIds" state before and still uses that. I just moved the check into its own function.